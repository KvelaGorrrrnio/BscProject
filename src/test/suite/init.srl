list list list int S

init S [1,2,10]

free S [1,2,10]

