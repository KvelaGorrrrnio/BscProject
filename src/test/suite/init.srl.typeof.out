S : list  list  list  int
n : int
